### PR TITLE
fix: Object examples not properly displayed as placeholders

### LINKED ISSUE #3951: The placeholder for the BeginDialogs options has a typo
## Describe the bug

If "Expression" is selected as the type for the begin dialog options field, it says "ex. [Object object]"

## Version

 1.1.0

Screen Shot 2020-08-31 at 4 06 17 PM

### PR DESCRIPTION
**Pull request by LouisEugeneMSFT**
## Description

Stringifying object examples

## Task Item

closes #3951

## Screenshots

image

### CODE REVIEW

File: Composer/packages/adaptive-form/src/utils/uiOptionsHelpers.ts [hunk 1]
```diff
@@ -46,7 +46,13 @@ export function getUiPlaceholder(props: FieldProps): string | undefined {
   } else if (placeholder) {
     fieldUIPlaceholder = placeholder;
   } else if (schema && Array.isArray(schema.examples) && schema.examples.length > 0) {
-    fieldUIPlaceholder = `ex. ${schema.examples.join(', ')}`;
+    const examplesStrings = schema.examples.map((example) => {
```
Comment (GeoffCoxMSFT) [inline Composer/packages/adaptive-form/src/utils/uiOptionsHelpers.ts:49]: Could you add a/update the unit test for this?  One of the goals for R11 is to increase test coverage.

Comment (LouisEugeneMSFT) [reply Composer/packages/adaptive-form/src/utils/uiOptionsHelpers.ts:49]: Done

**Comment by a-b-r-o-w-n** [comment]
There are some build errors now @LouisEugeneMSFT

File: Composer/packages/adaptive-form/src/utils/uiOptionsHelpers.ts [hunk 2]
```diff
@@ -47,7 +47,15 @@ export function getUiPlaceholder(props: FieldProps): string | undefined {
   } else if (placeholder) {
     fieldUIPlaceholder = placeholder;
   } else if (schema && Array.isArray(schema.examples) && schema.examples.length > 0) {
-    fieldUIPlaceholder = formatMessage('ex. { example }', { example: schema.examples.join(', ') });
+
+    const examplesStrings = schema.examples.map((example) => {
+      if (typeof example === 'object') {
```
Comment (hatpick) [inline Composer/packages/adaptive-form/src/utils/uiOptionsHelpers.ts:52]: FYI ```typeof null``` returns "object" too

**Review by GeoffCoxMSFT — APPROVED** [review]

**Pull request merged**